tools/perf/ResultsComparer/Bdn/BdnComparer.cs

```diff
@@ -0,0 +1,128 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Text.Json;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using ResultsComparer.Core;
+using Perfolizer.Mathematics.SignificanceTesting;
+using Perfolizer.Mathematics.Thresholds;
+
+namespace ResultsComparer.Bdn
+{
+    /// <summary>
+    /// Results comparer for BenchmarkDotNet full JSON reports.
+    /// </summary>
+    class BdnComparer : IResultsComparer

```

Comment:
Is this class internally scoped `private`? Would `internal` or `public` be a better scope?

Reply:
`class` defaults to `internal`